### PR TITLE
Make border size of PageNavigator tabs consistent

### PR DESCRIPTION
**Pull request by Willdotwhite**
Not sure if valid, but makes the nav bottom border match the selected tab border size 

image

### CODE REVIEW

File: src/components/PageNavigator.tsx [hunk 1]
```diff
@@ -6,15 +6,15 @@ const NavTab: React.FC<{to: string}> = ({children, ...props}) => (
...
+    activeStyle={{transform: "translate(0px, 2px)", borderWidth: "3px 3px 0 3px", background:"linear-gradient(#333, #000)"}}
     className="leading-tight text-2xl font-light bg-black py-3 px-4 w-48 text-white text-center uppercase whitespace-pre rounded-t-lg border-t border-l border-r"
   >
     {children}
   </NavLink>
 )
 
 export const PageNavigator: React.FC = () => (
-  <div className="flex flex-row justify-center items-center space-x-4 my-8 border-b border-white">
+  <div className="flex flex-row justify-center items-center space-x-4 my-8 border-b border-white" style={{borderBottomWidth: "2px"}}>
```
Comment (runny-yolk) [inline src/components/PageNavigator.tsx:17]: Remove the `style` attribute and just change `border-b` to `border-b-2` 👍

**Comment by runny-yolk** [comment]
FWIW, I think this PR would make the currently selected tab less clear, which defeats the point of why I gave it a thicker border in the first place - but I don't want to close this PR without outside input. @SlamBamActionman , thoughts?

**Comment by runny-yolk** [comment]
Screenshots:
Without this PR:
image

With this PR:
image

**Comment by SlamBamActionman** [comment]
If you want the line widths to match, you need 3px, not 2px.
On one hand I think it looks better visually, on the other I think it might make the top bar draw too much visual attention. I'd say not to implement this change.

**Comment by Willdotwhite** [comment]
Closing by consensus of "Unnecessary/unwanted change"

**Pull request closed without merging**